### PR TITLE
OSDOCS-10806 release note

### PR DESCRIPTION
**Pull request by jmanthei**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OSDOCS-10806

Link to docs preview: https://77601--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jmanthei** [comment]
@jinyunma Will this be sufficient or should I include a warning like in the other PR?

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -68,6 +68,12 @@ With this release, you can now install {op-system} to iSCSI boot devices. Multip
 [id="ocp-4-16-installation-and-update_{context}"]
 === Installation and update
 
+Ingress capability is now a configurable cluster capability. It must be enabled for OpenShift clusters to start.
```
Comment (jinyunma) [inline release_notes/ocp-4-16-release-notes.adoc:71]: The description looks confused here, do we need to mention that it is optional capability for hypershift, but always enabled in standalone OCP? 
@alebedev87 @lihongan  what's your opinion for this?

Comment (lihongan) [reply release_notes/ocp-4-16-release-notes.adoc:71]: I think ingress capability is not configurable in standalone OCP.
cc @melvinjoseph86

Comment (melvinjoseph86) [reply release_notes/ocp-4-16-release-notes.adoc:71]: yes, we need to mention this as option capability to Hypershift and always enabled for standalone OCP

**Comment by jmanthei** [comment]
@label peer-review-needed

**Pull request closed without merging**